cob_hardware_config/cob4-1/config/image_flip_params_sensorring_cam3d.yaml

```diff
@@ -1,30 +1,22 @@
 # rotation mode (0=fixed angle as defined  by rotation_angle, 1=automatic upright rotation against gravity using reference coordinate frame with upwards directed z-axis, 2=as 1 but image rotation only in 90deg steps -> faster)
 # int
-rotation_mode: 0
+rotation_mode: 1
```

Comment:
Here you have activated the computationally most heavy option. Are you sure to do this on default? This rotates the image in arbitrary directions. Do you have a camera that is not built in accurately enough?